docs/cql/describe-schema.rst

```diff
@@ -95,17 +95,17 @@ cqlsh support
     | ``DESCRIBE [FULL] SCHEMA WITH INTERNALS AND PASSWORDS`` | 6.0.23                    |
     +---------------------------------------------------------+---------------------------+
 
-Appendix I, salted hashes
--------------------------
+Appendix I, hashed passwords
+----------------------------
 
-A salted hash is an encrypted form of a password stored by ScyllaDB to authenticate roles. The statements returned
+A hashed password is an encrypted form of a password stored by ScyllaDB to authenticate roles. The statements returned
```

Comment:
I wonder if we shouldn't have left the word "salted" somewhere in this description.

Reply:
Personally, I'm against it for one reason: leaving the word out we give ourselves room for arbitrary changes in the semantics of it in the future. I know close to nothing about encryption, but I assume there are many other types of hashed passwords, so I'd keep the more general term. At the moment, we don't support any other use of it other than restoring the schema and roles, so it doesn't do any harm, while we can always make it more precise if need be